lib/hako/commander.rb

```diff
@@ -13,12 +13,30 @@ def initialize(app)
       @app = app
     end
 
+    def show_yaml(dry_run: false)
+      yaml = @app.yaml.dup
+      if dry_run
+        containers = load_containers('latest', dry_run: dry_run)
+        containers.each do |name, container|
+          if yaml.dig('additional_containers', name, 'env')
+            yaml.dig('additional_containers', name, 'env').merge!(container.env)
+          elsif yaml.dig(name, 'env')
+            yaml.dig(name, 'env').merge!(container.env)
+          end
+        end
+      end
+      yaml
+    end
+
     # @param [Boolean] force
     # @param [String] tag
     # @param [Boolean] dry_run
     # @return [nil]
     def deploy(force: false, tag: 'latest', dry_run: false, timeout:)
       containers = load_containers(tag, dry_run: dry_run)
+      if dry_run
+        print_expanded_yaml(containers)
```

Comment:
`print_expanded_yaml` doesn't seem to tell which provider expands the variable...
Could you show me example output?